tools/meta/tests/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ target_link_libraries(inviwo-meta-unittests PUBLIC inviwo::meta)
 target_link_libraries(inviwo-meta-unittests PUBLIC gtest)
 
 if(MSVC)
-    target_compile_options(inviwo-meta-unittests PRIVATE /W4 /utf-8 /permissive- /diagnostics:caret)
+    target_compile_options(inviwo-meta-unittests PRIVATE /W4 /utf-8 /openmp- /permissive- /diagnostics:caret)
```

Comment:
Think this is quite fragile it depends on /openmp- to removed the already added openmp from the target. 
It would be better to remove the global openmp flag in globalconfig.cmake and set them only on the targets needed in ivw_define_standard_properties